src/main/java/fr/insalyon/creatis/gasw/execution/GaswSubmit.java

```diff
@@ -101,17 +111,23 @@ protected String generateScript() throws GaswException {
      * @return Name of the script file.
      */
     private String publishScript(String symbolicName, String script) {
+        String fileName = null;
 
         try {
             File scriptsDir = new File(GaswConstants.SCRIPT_ROOT);
             if (!scriptsDir.exists()) {
                 scriptsDir.mkdir();
             }
 
-            String fileName = symbolicName.replace(" ", "-");
-            fileName += "-" + System.nanoTime() + ".sh";
-            writeToFile(GaswConstants.SCRIPT_ROOT + "/" + fileName, script);
-
+            if (gaswInput.isMoteurLiteEnabled()) {
+                fileName = gaswInput.getJobId(); 
+                writeToFile(GaswConstants.SCRIPT_ROOT + "/" + fileName, script);
```

Comment:
I feel like in the moteurlite case we write the script file twice : 1 here and 1 in MoteurLiteScriptGenerator
Is it twice the same location or 2 different locations ?